projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/class-verbum-comments.php

```diff
@@ -272,7 +276,7 @@ public function comment_form_defaults( $args ) {
 				'logged_in_as'         => '',
 				'comment_notes_before' => '',
 				'comment_notes_after'  => '',
-				'title_reply'          => __( 'Leave a Reply', 'jetpack-mu-wpcom' ),
+				'title_reply'          => __( 'Leave a comment', 'jetpack-mu-wpcom' ),
```

Comment:
I updated this to `Leave a comment` according to @crisbusquets's comment: p1706631945152779/1706626540.417219-slack-C02T4NVL4JJ